static/docs/user-guide/running-dvc-on-windows.md

```diff
@@ -47,3 +47,26 @@ The performance of NTFS degrades while handling large volumes of files in a
 directory.
 [Here](https://stackoverflow.com/questions/197162/ntfs-performance-and-large-volumes-of-files-and-directories)
 is the resource for reference.
+
+## Enabling paging with `less`
+
+By default DVC uses `less` to page output for some commands. Although, many
+operating systems have `less` preinstalled, there are still platforms where
+`less` is not installed. This usually happens for Microsoft Windows.
+Fortunately, there is a easy way of installing `less` via
+[Chocolatey](https://chocolatey.org/).
+
+```dvc
+$ choco install less
+```
+
+If you do not want to use [Chocolatey](https://chocolatey.org/) for some reason,
+you can still provide `less` by your own way. However, you must make `less` to
+be runnable from `cmd`/`Powershell`. This usually means to add the `less`
+installation directory to the `PATH` environement vairable.
+
+### Providing a custom pager
+
+See
+[this section](/doc/command-reference/pipeline/show#providing-a-custom-pager)
+for details.
```

Comment:
I don't think any of this is necessary in this doc 🙂 But thanks!

Reply:
too late ;)

should i remove it? :)

Reply:
I think at least the last section isn't necessary given https://github.com/iterative/dvc.org/pull/831#discussion_r356804261.

The top paragraph here I understand what it's for but maybe that is just an obvious conclusion people can come to. Also it has some typos. If you feel strongly about making a note about `less` in `PATH`, maybe just rephrase it into a shorter note (can probably be a single sentence) in an md blockquote `>` ? Thanks

Reply:
I'll remove the whole "Providing a custom pager" section

Reply:
removed thanks

Reply:
Np but missing revision for the other paragraph. I'll make a separate comment, thanks.